test/test_other.py

```diff
@@ -9753,6 +9753,12 @@ def check_source_map_loc_info(address, loc):
           stdout=PIPE).stdout
       self.assertIn(loc, out)
 
+    def check_func_info(address, func):
```

Comment:
All the tests except the last one I added in this PR share the results of the get_addr calls, which means that if we did that, we'd run objdump more times and do more work overall. Would that be worth it?

Reply:
I don't have any great ideas about how to simplify, but this test is IMHO way to long already and can fail in way to many ways.

I just tried to read and understand what was going in this test and found it pretty hard to figure out what was even going on.   Perhaps we can find some way to refactor in the future?

I guess in normal python unittest setup you would make emsymbiler test into its own class with a setUpClass method.. but that doesn't really work with the way we run out tests.

Reply:
I actually realized that llvm-symbolizer doesn't work correctly for object file symbol tables, so I removed the test for that (that was 5). I split the tests using exact source locations apart from the ones using functions only (currently it's just the name section one, but if we fix object files, it would go there too). There's a little duplication, but tests are a little simpler.